### PR TITLE
Added handler for outside clicking

### PR DESCRIPTION
**Pull request by prasarCodes**
## Changes proposed
The handler will enable clicking outside the profile picture button to close the popover

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: src/lib/components/profile-picture.svelte [hunk 1]
```diff
@@ -42,6 +42,23 @@
...
+      if (!root.contains(target) && !popover.contains(target)) {
+        displayPopover = false;
+        destroyInstance();
+      }
+    }
+  }
+
+  onMount(() => {
+    window.addEventListener('click', handleOutsideClick);
+    onCleanup(() => {
```
Comment (Cahllagerfeld) [inline src/lib/components/profile-picture.svelte:58]: I think the Cleanup-Hook doesn't exist.
You can return a function from `onMount`, that is run to clean up, see the [docs](https://svelte.dev/tutorial/onmount).

**Review by Cahllagerfeld — CHANGES_REQUESTED** [review]
Other than that it looks good to me 👍

File: src/lib/components/profile-picture.svelte (hunk 1 above)
Comment (prasarCodes) [reply src/lib/components/profile-picture.svelte]: Hi! Thanks for your input.
I've removed the onCleanup hook and added a return statement.
Please review the code!

**Review by Cahllagerfeld — APPROVED** [review]
Nice work!

**Pull request merged**